### PR TITLE
changes made

### PR DESCRIPTION
**Pull request by jllrk1**

### CODE REVIEW

**Comment by jllrk1** [comment]
changes made

File: script.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
+$(function() {
+    var $h2 = $('h2');
+    var $mPage = $('#mPage');
+    $h2.on("click", function(){
+    $mPage.after('<div class = "favItems"></div>');
+        $h2.remove();
+        $mPage.append('<h3>Although it is difficult to narrow it down, here are just a few of my favorite things</h3>');
+        var $favItems = $('<h4>Camping</h4><img src="camping_fullsize_story1.jpg"><h4>Music</h4><img src="infra.jpg"><h4>Snowboarding</h4><img src="12-2012-Swanson-8.jpg"><h4>Disc Golf</h4><img src="IMG_0954.jpg"><h4>Football</h4><img src="nfl.jpg">');
```
Comment (mbMosman) [inline script.js]: You should always declare the variables at the top of their scope (start of the function).

File: script.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+
```
Comment (mbMosman) [inline script.js]: The code would be more readable if it was indented consistently.  Try the Beautify plugin/extension for Brackets.

Comment (jllrk1) [inline script.js]: How do I get that beautify plugin/extension?

**Pull request closed without merging**